runtime/src/snapshot_minimizer.rs

```diff
@@ -587,8 +619,12 @@ mod tests {
         }
 
         assert_eq!(minimized_account_set.len(), 6);
-        let minimizer =
-            SnapshotMinimizer::new(bank, current_slot, current_slot, minimized_account_set);
+        let minimizer = SnapshotMinimizer {
```

Comment:
sorry - The use of 'new()' did result in reduced/single line construction for test code ;-) I didn't forsee that. You could have kept it as a test function to make tests easier to write. You can do an impl in the test module that is only visible there. Like this is fine, too. However you prefer it... Was just trying to keep the set of code that requires testing to a minimum.
To be clear - this is fine as it is or you can revert this back to using new for test code now, in a different pr, or never. I'd probably vote for never. ;-)